test/TestProjects/Customizations-Cadl/Customizations/RootModel.cs

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace CustomizationsInCadl
```

Comment:
I didn't see this case in the doc, do we need to support this?

Reply:
It is not a customization. This model is needed to reference all other models so that they are correctly referenced. Since `ModelToMakeInternal` customization changes accessibility of the model but not of its usages, we need to update it manually.